modules/update-service-create-service-cli.adoc

```diff
@@ -0,0 +1,80 @@
+[id="update-service-create-service-cli_{context}"]
+= Creating an OpenShift Update Service application by using the CLI
+
+You can use the {product-title} CLI to create an OpenShift Update Service application.
```

Comment:
Can you please replace all instances of "{product-title} CLI" with "OpenShift CLI (`oc`)" ?